### PR TITLE
Create .replit

### PR DESCRIPTION
**Pull request by alexyy802**
## Summary

Made .replit for repl.it tutorial file

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

File: examples/replit/.replit [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language="python3" 
+run = """
+python main.py #You can change main.py into something else just make sure its the main file
```
Comment (VincentRPS) [inline examples/replit/.replit:3]:
```suggestion
python main.py # You can change main.py into something else just make sure its the main file
```

**Review by VincentRPS — CHANGES_REQUESTED** [review]

**Review by VincentRPS — APPROVED** [review]
lgtm @Hype3808 will need to look aswell

**Comment by alexyy802** [comment]
> lgtm @Hype3808 will need to look aswell

yea :/

**Review by Hype3808 — CHANGES_REQUESTED** [review]
Add `pip install -U texus` before `python main.py`

**Comment by alexyy802** [comment]
> Add `pip install -U texus` before `python main.py`

Aight

**Comment by alexyy802** [comment]
@Hype3808 Done

**Review by Hype3808 — APPROVED** [review]

**Pull request merged**